data/human/free worlds 4 epilogue.txt

```diff
@@ -25,7 +25,8 @@ mission "FW Epilogue: Ijs and Katya"
 				`	(Not right now.)`
 					defer
 			`	While looking Ijs up in the campus directory, you notice something else: Katya is listed in the directory too, as "Visiting Professor of Civics and Economics." You give Ijs a call and ask if he and Katya would like to meet up with you for dinner, and they readily agree.`
-			`	When you join them at the restaurant, Katya says, "It's really great to see you again, <first>."`
+			`	You follow their directions to a campus restaurant which still seems to be under construction. Katya eyes one of the less stable-looking walls suspiciously, but Ijs assures both of you that it's perfectly safe and the restaurant is fully operational despite the ongoing work.`
```

Comment:
```suggestion
			`	You follow their directions to a campus restaurant which seems to still be under construction. Katya eyes one of the less stable-looking walls suspiciously, but Ijs assures both of you that it's perfectly safe and the restaurant is fully operational despite the ongoing work.`
```


Subtle but important difference. The original implies having seen it seemingly under construction before and it still is now, suggestion implies only that construction is seemingly ongoing (as opposed merely to repair/maintenance work).